### PR TITLE
bring in Dassets; add Railtie that configures/initializes Dassets

### PR DESCRIPTION
**Pull request by kellyredding**
This aliases Dassets as MuchRails::Assets as is the convention
for dependencies that MuchRails brings in. This also adds a Railtie
and configuration method for configuring Dassets in a default
manner. This is an opinionated default. In the future, we'll most
likely add something like profiles or provide a way to manually
configure source behavior.

### CODE REVIEW

File: lib/much-rails/railtie.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module MuchRails
+  class Railtie < Rails::Railtie
+    initializer "much-rails-gem" do |app|
+      require "much-rails/assets"
```
Comment (kellyredding) [inline lib/much-rails/railtie.rb:6]: @nweathersbee note: I didn't require assets in the main MuchRails require file as I want this to be somewhat optional/late-bound. This says, there is no point requiring the assets stuff if you aren't using the Railtie. FYI.

File: Gemfile [hunk 2]
```diff
@@ -5,3 +5,6 @@ ruby "~> 2.5"
 gemspec
 
 gem "pry", "~> 0.12.2"
+gem "dassets", path: "/Users/kelly/projects/redding/gems/dassets"
+gem "dassets-erubi", path: "/Users/kelly/projects/redding/gems/dassets-erubi"
+gem "dassets-sass", path: "/Users/kelly/projects/redding/gems/dassets-sass"
```
Comment (nweathersbee) [inline Gemfile]: 👀

**Review by nweathersbee — APPROVED** [review]
Other than the local gem paths, LGTM!

**Review by jcredding — APPROVED** [review]

File: Gemfile (hunk 2 above)
Comment (nweathersbee) [reply Gemfile]: nvm, I see what you did here. :)

**Pull request merged**